code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -174,6 +174,43 @@
 	body_parts_covered = CHEST|ARMS|GROIN|LEGS|FEET
 	flags_inv = HIDESHOES|HIDEJUMPSUIT
 
+//I will ensure that wearing this will warent a sufficient beating to take it off again
+/obj/item/clothing/head/chungus
+	name = "chungus helmet"
+	desc = "BIG CHUNGUS HUER DU UUUUUUUUUUUUUUUU CHUNGUS FUNNY REDDIT MOMENT BIG GROBE CHUNGUS KIONO REEVES WHOLE SOME 100 ELON MUSK."
```

Comment:
please dude, at least try

Reply:
There is no way you wont get beaten up wearing this